korlibs-datastructure/src/korlibs/datastructure/_Datastructure_event.kt

```diff
@@ -88,7 +87,7 @@ open class SyncEventLoop(
             if (running) {
                 timedTasks.add(task)
             } else {
-                Console.warn("WARNING: QUEUED TASK time=$time interval=$interval without running")
+                //TODO: add back logger Console.warn("WARNING: QUEUED TASK time=$time interval=$interval without running")
```

Comment:
This class/package should be kept in the korge-foundation for now, and there the logger package should be available.